tests/monitoring_integration_test.py

```diff
@@ -144,6 +145,22 @@ def testShippingServiceDashboard(self):
 		found_dashboard = self.checkForDashboard('Shipping Service Dashboard')
 		self.assertTrue(found_dashboard)
 
+	def testLogBasedMetricDashboard(self):
+		""" Test that the Log Based Metric Dashboard gets created. """
+		found_dashboard = self.checkForDashboard('Log Based Metric Dashboard')
+		self.assertTrue(found_dashboard)
+
+class TestLogBasedMetric(unittest.TestCase):
+	def setUp(self):
+		self.client = logging_v2.MetricsServiceV2Client()
+		self.project_id = getProjectId()
+
+	def testCheckoutServiceLogMetric(self):
+		""" Test that the log based metric for the Checkout Service gets created. """
+		metric_name = self.client.metric_path(self.project_id, "checkoutservice_log_metric")
+		response = self.client.get_log_metric(metric_name)
```

Comment:
is there a way to test metric type (alignment and reduction types you decided to use in terraform)?

Reply:
The only metric types we can test are metric_kind and value_type but they are ENUMs so the tests will look a little hard coded. I don't think it's too important to test it - but let me know if you think otherwise.